docs/api-markdown-documenter/render-api-documentation.js

```diff
@@ -86,7 +85,7 @@ async function renderApiDocumentation() {
 		throw error;
 	}
 
-	buildNavBar(documents);
+	await buildNavBar(documents);
```

Comment:
Nit: if we move this down below the `console.groupEnd();` call below, we could wrap it in its own logging group prefaced with something like `console.log("Generating nav contents...");` Might make debugging future issues a bit easier to track down.